client/registry.go

```diff
@@ -0,0 +1,309 @@
+package client
+
+import (
+	"sync"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/mysteriumnetwork/payments/bindings"
+)
+
+type HermesImplementationRegistry struct {
+	callers     map[string]*bindings.HermesImplementationCaller
```

Comment:
Also this should be private no? Its useless as public now

Reply:
Fixed

Reply:
feeling capitalist today, privatize everything!